unitTests/ui/ResultLinkTest.ts

```diff
@@ -23,531 +23,544 @@ export function ResultLinkTest() {
       return fakeResult;
     }
 
-    function initResultLink() {
-      test = Mock.advancedResultComponentSetup<ResultLink>(ResultLink, fakeResult, undefined);
+    function initResultLink(options?: Mock.AdvancedComponentSetupOptions) {
+      test = Mock.advancedResultComponentSetup<ResultLink>(ResultLink, fakeResult, options);
     }
 
     function htmlContainingXSS() {
       return '<IMG SRC=/ onerror="alert(String.fromCharCode(88,83,83))"></img>';
     }
 
-    beforeEach(() => {
-      fakeResult = buildFakeResult();
-      initResultLink();
-      spyOn(test.cmp, 'openLink');
-      spyOn(window, 'open');
-    });
-
-    afterEach(function () {
-      test = null;
-      fakeResult = null;
-    });
-
-    it('should have its tabindex value set to 0', () => {
-      expect(test.cmp.element.getAttribute('tabindex')).toBe('0');
-    });
-
-    it('should have an aria-label to prevent screen readers from reading the #title attribute', () => {
-      expect(test.cmp.element.getAttribute('aria-label')).toBe('Result');
-    });
-
-    describe('with global result link options', () => {
-      function createEnvWithGlobalResultLinkOptions() {
-        const searchInterface = test.env.searchInterface;
-        searchInterface.options.originalOptionsObject = {
-          [ResultLink.ID]: {
-            alwaysOpenInNewWindow: true
-          }
-        };
-
-        return new Mock.MockEnvironmentBuilder().withResult(fakeResult).withSearchInterface(searchInterface);
-      }
-
-      it('the global options are used', () => {
-        const env = createEnvWithGlobalResultLinkOptions();
-        const cmp = new ResultLink(
-          env.getBindings().element,
-          {},
-          (env.getBindings() as any) as IResultsComponentBindings,
-          env.result,
-          env.os
-        );
-        expect(cmp.options.alwaysOpenInNewWindow).toBe(true);
+    describe('with a tabindex', () => {
```

Comment:
I recommend hiding whitespace to see the actual difference:
![image](https://user-images.githubusercontent.com/54454747/157683065-6302d09e-69b2-419c-84c5-f04ae3aed20f.png)